src/data.js

```diff
@@ -1,6 +1,6 @@
 var covers = [
   './assets/bluebrocade.jpg',
-  './assets/dance.jpg', 
```

Comment:
Same as in index.html, git thinks a change happened here but both versions look identical to me.